src/records.jl

```diff
@@ -98,7 +141,7 @@ struct DefaultRecord <: Record
 end
 
 """
-    DefaultRecord(name::AbstractString, level::AbstractString, msg::AbstractString)
+    DefaultRecord(name::AbstractString, level::AbstractString, levelnum::Int, msg::AbstractString)
```

Comment:
I think in the future we'll want to just have a global level dict that we can lookup rather than supporting arbitrary levels.

Reply:
Yes, but you wanted that in a separate PR.

Reply:
Oh, duh, I missed that this was just updating the docstring to fit the current args.